src/runtime/GlobalObject.h

```diff
@@ -604,6 +608,16 @@ class GlobalObject : public Object {
         return m_weakSetPrototype;
     }
 
+    FunctionObject* generator()
+    {
+        return m_generator;
```

Comment:
It seems that `generator()` function is unnecessary.

Reply:
Sure!

Reply:
IMO `m_generator` also looks unnecessary because generator functions are defined only by `GeneratorFunction` constructor or function* expression. So, `m_generator` (GlobalObject.Generator) can never be used.

Reply:
I would not remove it until the whole feature is not finished yet. I'll add a note to remove it if it is possible.